README.md

```diff
@@ -6,35 +6,43 @@ For the last decade, we have been accumulating the knowledge and practicalities
 
 This Handbook offers us an opportunity to clarify what Niteo is and how we operate. It allows greater transparency to our partners, potential new team members, and the wider Open Source community.
 
-## Company
-
-* [About Niteo - Vision & Values](https://github.com/niteoweb/handbook/blob/master/niteo.md)
-* [History](https://github.com/niteoweb/handbook/blob/master/history.md)
-* [Who Does What](https://github.com/niteoweb/handbook/blob/master/who-does-what.md)
-
-## People
-* [Onboarding](https://github.com/niteoweb/handbook/blob/master/onboarding.md)
-* [Profit Sharing](https://github.com/niteoweb/handbook/blob/master/profit-sharing.md)
-* [Benefits](https://github.com/niteoweb/handbook/blob/master/benefits.md)
-* [Salary](https://github.com/niteoweb/handbook/blob/master/salary.md)
-* [Catchup Meetings](https://github.com/niteoweb/handbook/blob/master/catchup-meetings.md)
-
-## Work
-* [How We Work](https://github.com/niteoweb/handbook/blob/master/how-we-work.md)
-* [Daily Standup](https://github.com/niteoweb/handbook/blob/master/standup.md)
-* [Using Slack](https://github.com/niteoweb/handbook/blob/master/using-slack.md)
-* [Security Policy](https://github.com/niteoweb/handbook/blob/master/security.md)
-* [Managing Docs](https://github.com/niteoweb/handbook/blob/master/managing-docs.md)
-* [Work Process](https://github.com/niteoweb/handbook/blob/master/work-process.md)
-* [Glossary](https://github.com/niteoweb/handbook/blob/master/glossary.md)
-* [Find Your Next Task](https://github.com/niteoweb/handbook/blob/master/next-task.md)
-
-## Support
-* [Support Documentation](https://github.com/niteoweb/support)
-* [Everyone On Support](https://github.com/niteoweb/handbook/blob/master/everyone-on-support.md)
-* [Handling Support Requests](https://github.com/niteoweb/handbook/blob/master/support.md)
-
-## Handbook Contribution
+## About the company
+
+* [About Niteo](company/niteo.md) - The vision and values.
+* [History](company/history.md) - How we started and have evolved.
+* [Who does what](company/who-does-what.md)
+
+## For the people
+* [Profit Sharing](people/profit-sharing.md)
+* [Benefits](people/benefits.md)
+* [Salary](people/salary.md)
+* [Catchup Meetings](people/catchup-meetings.md)
+
+## Nitean roles
+* [Onboarding](roles/onboarding.md)
+* [Support](roles/support.md)
+* [Everyone On Support](roles/everyone-on-support.md)
```

Comment:
Shouldn’t this be under `Working process` since it’s a regular monthly operations for the team?

Reply:
Sure it can be linked to from working process but it is a role that Niteans undertake